packages/@okta/vuepress-site/guides/sign-into-spa/section06/index.md

```diff
@@ -1,9 +1,42 @@
 ---
 title: Configure the SDK
 ---
-## Configure the SDK
 
-Initialize and configure the Okta SDK using the config object you created in the previous step. 
+You'll need two values from the Okta Application and the Developer Console you worked with in [Create an Okta Application](todo):
+
+* **Client ID** - Find it on the Okta Application details (General tab).
```

Comment:
This is no longer clear as to where to find the client id. They can either get it by looking at the applications list or by going to the General tab of the specific application. I don't think this change is more clear. How about:

* **Client ID** - Find it in the applications list or on the application's **General** tab.